src/objects.js

```diff
@@ -1,41 +1,59 @@
 const createPerson = (name, age) => {
-  // your code here
+return {
+  name: name,
+  age: age,
+
+};
 };
 
 const getName = object => {
-  // your code here
+return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+return object.hasOwnProperty;
 };
 
 const isOver65 = person => {
-  // your code here
+if (person.age > 65) {
+  return true;
+} else {
+  return false;
+}
 };
```

Comment:
as previously mentioned, just return the condition rather than checking the condition and returning `true` or `false`, i.e.
```
const isOver65 = person => {
  return person.age > 65
};
```